grafana-plugin/src/utils/index.ts

```diff
@@ -95,3 +96,10 @@ export function getPaths(obj?: any, parentKey?: string): string[] {
 export function pluralize(word: string, count: number): string {
   return count === 1 ? word : `${word}s`;
 }
+
+export function isUseProfileExtensionPointEnabled(): boolean {
+  const { major, minor } = getGrafanaVersion();
+  const isRequiredGrafanaVersion = major > 10 || (major === 10 && minor >= 3);
+
+  return isRequiredGrafanaVersion;
+}
```

Comment:
this seems duplicate of [this function](https://github.com/grafana/oncall/pull/3296/files#diff-bc9e12d72092a3f2944bb5bb941ee47f12f58c37e4dcdfbfd667cac49b83876bR35-R45) defined in `module.ts`

Reply:
Yea this one is actually tricky because it adds a loophole if you reference stuff in module.ts from my attempts

Reply:
Maybe extracting that to separate file and importing it from both `module.ts` and this file will help with circular dep?

Reply:
You still need to reference the module file back because there's the plugin defined - for this line `'configureExtensionComponent' in plugin`